qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -29,8 +28,7 @@
                                    store_credentials)
 from .credentials.updater import update_credentials
 from .exceptions import IBMQAccountError, IBMQApiUrlError, IBMQProviderError
-from .ibmqprovider import IBMQProvider
-from .utils.deprecation import deprecated, UPDATE_ACCOUNT_TEXT
```

Comment:
The `deprecated` function in `ibmq/utils/deprecation.py` does not seem like it is used anymore. It also might be worth removing, unless there is an intended use for it?

Reply:
Spot on! The plan is to remove it during the follow-up PRs, along with more code that will become unused after the changes.